--- src/libstore/local-store.cc	1969-12-31 16:00:01.000000000 -0800
+++ src/libstore/local-store-fixed.cc	2018-09-04 22:20:51.312105000 -0700
@@ -461,7 +461,7 @@
         for (auto & eaName: tokenizeString<Strings>(std::string(eaBuf.data(), eaSize), std::string("\000", 1))) {
             /* Ignore SELinux security labels since these cannot be
                removed even by root. */
-            if (eaName == "security.selinux") continue;
+            if (eaName == "security.selinux" || eaName == "system.nfs4_acl") continue;
             if (lremovexattr(path.c_str(), eaName.c_str()) == -1)
                 throw SysError("removing extended attribute '%s' from '%s'", eaName, path);
         }
