--- /global/home/users/jefdaj/nix-boot/nix-1.11.9/local.mk	1969-12-31 16:00:01.000000000 -0800
+++ nix-local.mk	2017-06-10 22:20:57.678085000 -0700
@@ -10,6 +10,8 @@
 GLOBAL_CXXFLAGS += -I . -I src -I src/libutil -I src/libstore -I src/libmain -I src/libexpr \
   -Wno-unneeded-internal-declaration
 
+GLOBAL_LDFLAGS += -pthread
+
 $(foreach i, config.h $(call rwildcard, src/lib*, *.hh) src/nix-store/serve-protocol.hh, \
   $(eval $(call install-file-in, $(i), $(includedir)/nix, 0644)))
 
